### PR TITLE
Update Feature Primitives page to use woodwork

### LINKED ISSUE #1536: Update Feature primitives Page with Woodwork Dataframe
- The page below should be updated with the new methods to add DataFrames to an Entityset
  - https://featuretools.alteryx.com/en/latest/getting_started/primitives.html
- We should also at the same time **use Jupyter notebooks, rather than RST files.**

### PR DESCRIPTION
**Pull request by frances-h**
Updates the `primitives.ipynb` to use the woodwork-integrated EntitySets and DFS calls.

Closes #1536

### CODE REVIEW

File: docs/source/getting_started/primitives.ipynb [hunk 1]
```diff
@@ -79,33 +76,50 @@
...
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Aggregation vs Transform Primitive\n",
+    "\n",
+    "In the example above, we use two types of primitives.\n",
+    "\n",
+    "**Aggregation primitives:** These primitives take related instances as an input and output a single value. They are applied across a parent-child relationship in an entity set. E.g: `\"count\"`, `\"sum\"`, `\"avg_time_between\"`."
```
Comment (thehomebrewnerd) [inline docs/source/getting_started/primitives.ipynb:87]: `entity set` -> `EntitySet`?

File: docs/source/getting_started/primitives.ipynb [hunk 2]
```diff
@@ -179,14 +190,14 @@
...
   {
-   "cell_type": "raw",
+   "cell_type": "markdown",
    "metadata": {
-    "raw_mimetype": "text/restructuredtext"
+    "raw_mimetype": "text/markdown"
    },
    "source": [
-    "Above, we created a new transform primitive that can be used with Deep Feature Synthesis by deriving a new primitive class using :class:`TransformPrimitive` as a base and overriding :meth:`get_function` to return a function that calculates the feature. Additionally, we set the input data types that the primitive applies to and the return data type.\n",
+    "Above, we created a new transform primitive that can be used with Deep Feature Synthesis by deriving a new primitive class using `TransformPrimitive` as a base and overriding `get_function` to return a function that calculates the feature. Additionally, we set the input data types that the primitive applies to and the return data type.\n",
```
Comment (thehomebrewnerd) [inline docs/source/getting_started/primitives.ipynb:198]: Maybe worth a short mention that input an return types are defined by using a Woodwork ColumnSchema to specify the logical type and/or semantic tags that are valid for input/output. Don't need a lot of detail, maybe just a sentence and then a link to the Woodwork types and tags documentation: https://woodwork.alteryx.com/en/stable/guides/understanding_types_and_tags.html

**Review by thehomebrewnerd — COMMENTED** [review]
Just a couple minor suggestions here.

**Review by jeff-hernandez — APPROVED** [review]
Looks good!

**Pull request merged**